### ISSUE TITLE
'make audit' fails on IPv4 test

### ISSUE THREAD

**Issue by jfredrickson5** [cloud-gov/caulking#38]
I performed a new install of caulking and proceeded to run `make audit`, which had one failure:

```
 ✗ leak prevention catches IPv4 address in test repo
   (in test file caulked.bats, line 56)
     `[ ${status} -eq 1 ]' failed
   Initialized empty Git repository in /private/var/folders/j7/qqyv0gtj4rj5nl0_jr2sgh5c0000gn/T/gittest.JBJs85/.git/
```

test_helper.sh uses a private IP address range in the test, line 86:
```
Host: 10.20.30.40
```

If private ranges are supposed to be allowed in commits, then the test should be changed to not use a private range (e.g., 11.22.33.44 instead of 10.20.30.40). On the other hand, if private ranges should be considered secrets, then probably the regexes in local.toml need to be updated to catch them.

**Comment by onelittlebecca** [comment]
Tagged you in a potentially helpful thread on slack. Potentially being the keyword. 
If not, happy to help troubleshoot things with you

**Comment by jfredrickson5** [comment]
Fixed in #39
